wagtailio/project_styleguide/templates/patterns/includes/header.html

```diff
@@ -13,12 +13,24 @@
 
 
     <div class="header__col-two">
-        <button class="header__button header__button--search" aria-label="Open the search" data-site-search>
+        {# Desktop Search #}
```

Comment:
Note: on mobile when on search_results.html, I think this mobile view breaks the normal use case as the search results block is already shown, what do you think?

Reply:
I see what you mean - there's a double search of sorts. I don't think it's too much of an issue having thought about it. If a user on mobile landed on the search results page then they'd probably use the search input that on the page template but I don't think it's an issue if they opened the other search and used that instead.